src/shop.js

```diff
@@ -436,14 +436,20 @@ function BinartaShopjs(checkpoint, deps) {
                 fsm.persist(ctx);
                 shop.localStorage.setItem('binartaJSPaymentProvider', provider)
             };
-            if (!fsm.getPaymentProvider())
-                if (shop.localStorage.getItem('binartaJSPaymentProvider'))
+            
+            if (!fsm.getPaymentProvider()) {
+                if (shop.localStorage.getItem('binartaJSPaymentProvider')){
                     fsm.setPaymentProvider(shop.localStorage.getItem('binartaJSPaymentProvider'));
-                else {
+                } else {
                     var profile = application.profile();
-                    if (profile.availablePaymentMethods && profile.availablePaymentMethods.length == 1)
-                        fsm.setPaymentProvider(profile.availablePaymentMethods[0]);
+                    if (profile.availablePaymentMethods && profile.availablePaymentMethods.length == 1) {
+                      fsm.setPaymentProvider(profile.availablePaymentMethods[0]);
+                    } else if (profile.availablePaymentMethods && profile.availablePaymentMethods.length === 2 
```

Comment:
This is the actual change right? Test if paypal-classic is an option and then select it.

Reply:
correct.